sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogSuite.scala

```diff
@@ -40,6 +40,15 @@ abstract class ExternalCatalogSuite extends SparkFunSuite with BeforeAndAfterEac
 
   protected def resetState(): Unit = { }
 
+  // Clear the state before all tests
```

Comment:
Yeah, it connects to the same metastore. We can change it by setting `HiveConf.ConfVars.METASTORECONNECTURLKEY.varname` when creating a new external catalog.